### DISCUSSION TITLE
[Script request] Inventree

### DISCUSSION THREAD

**Discussion by SantiDev-web** [community-scripts/ProxmoxVE#981] [Request script] [answered]
### Application Name

Inventree

### Website

https://inventree.org/

### Description

InvenTree is an open-source inventory management system which provides intuitive parts management and stock control. A wide range of features makes InvenTree the perfect choice for businesses and hobbyists alike.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by 391a** [comment]
Duplicate. Use search function before.
#622

**Comment by tremor021** [comment, marked as answer]
Done here #2890
